linux/sys/sys_renameat2.c

```diff
@@ -0,0 +1,14 @@
+#include "sys/sys.h"
+
+#include <unistd.h>
+
+int
+sys_renameat2(struct LFILinuxThread *t, int olddir, lfiptr oldpathp, int newdir,
+    lfiptr newpathp, unsigned int flags)
+{
+  if (flags != 0){
+    return -LINUX_EINVAL;
+  }
+
+  return sys_renameat(t, olddir, oldpathp, newdir, newpathp);
+}
```

Comment:
Can you make sure that the formatting in this file is consistent with the rest of the codebase: indentation should have 4 spaces and there should be a space before the `{` in `flags != 0){`.